elementary/config/config.py

```diff
@@ -166,31 +162,31 @@ def has_gcs(self):
         return self.gcs_bucket_name and self.has_gcloud
 
     def validate_monitor(self):
-        self._validate_elementary_profile()
+        self._validate_internal_dbt_project()
         self._validate_timezone()
         if not self.has_slack:
             raise InvalidArgumentsError(
                 "Either a Slack token and a channel or a Slack webhook is required."
             )
 
     def validate_report(self):
-        self._validate_elementary_profile()
+        self._validate_internal_dbt_project()
```

Comment:
Maybe instead of running it for each method (and remember to add it to a new method), we will run it on the init?
That way no matter what, before we start anything we will validate the internal project?

Reply:
I thought so as well.
The problem is that we also use it [here](https://github.com/elementary-data/elementary/blob/e52e384ad264c43ebcab7221edf2eeec64374526/elementary/cli/cli.py#L46) and we don't want to make that validation there.

Reply:
We could also make it validate by default and only opt-out there.